src/Tool/Command/PhiveInstallCommand.php

```diff
@@ -0,0 +1,35 @@
+<?php declare(strict_types=1);
+
+namespace Zalas\Toolbox\Tool\Command;
+
+use Zalas\Toolbox\Tool\Command;
+
+final class PhiveInstallCommand implements Command
+{
+    private $alias;
+    private $bin;
+    private $sig;
+
+    public function __construct(string $alias, string $bin, ?string $sig = null)
+    {
+        $this->alias = $alias;
+        $this->bin = $bin;
+        $this->sig = $sig;
+    }
+
+    public function __toString(): string
+    {
+        $home = \dirname($this->bin);
+        $tmp = '/tmp/'.\md5($this->alias);
+
+        return \sprintf(
+            'phive --no-progress --home %s install %s %s -t %s && mv %s/* %s',
+            $home,
+            $this->sig ? '--trust-gpg-keys '.$this->sig : '--force-accept-unsigned',
```

Comment:
Are signatures not available for some packages?

Reply:
Yep. Some projects either don't sign or don't post the signature in their github releases page.

`phive` tries to be ubiquitous so it works even if your project doesn't have anything to do with `phive`, as long as you have semver releases containing `.phar` + `.phar.asc` files.

Reply:
> Starts looking good 
> 
> Builds are currently failing due to an issue unrelated to this PR: [symplify/symplify#3056](https://github.com/symplify/symplify/issues/3056)

I've noticed and concluded that it's external.

Should I rebase from master? Or wait for a fix?

Reply:
Up to you. We won't be able to merge until the build is fixed.

If it takes too long to fix ecs, I'll disable it for now.

Reply:
No rush.